### PR TITLE
update homestead block number in test fillers

### PR DESCRIPTION
**Pull request by winsvega**

### CODE REVIEW

File: test/libethereum/StateTestsFiller/Homestead/stSpecialTestFiller.json [hunk 1]
```diff
@@ -266,237 +266,13 @@
         }
     },
 
-    "block504980" : {
```
Comment (chriseth) [inline test/libethereum/StateTestsFiller/Homestead/stSpecialTestFiller.json:269]: Why was that removed?

Comment (winsvega) [inline test/libethereum/StateTestsFiller/Homestead/stSpecialTestFiller.json:269]: it's a dublicate of 
https://github.com/ethereum/libethereum/blob/develop/test/libethereum/StateTestsFiller/stSpecialTestFiller.json#L269

block 504980 is not of homestead
